spec/ddtrace/contrib/active_support/notifications/subscription_spec.rb

```diff
@@ -75,7 +75,7 @@
         end
 
         it 'sets span in payload' do
-          expect { subject }.to change { payload[:datadog_span] }.to be_instance_of(Datadog::Span)
+          expect { subject }.to change { payload[:datadog_span] }.to be_instance_of(Datadog::SpanOperation)
         end
```

Comment:
Minor: Should we test the actual instance here, rather than testing its class?

Reply:
As in not do `be_instance_of`? Can you clarify?

Reply:
I meant, check that the contents of the span (or that it is == to our double above), rather than just its class.